test/jest/Inputfelt.test.js

```diff
@@ -0,0 +1,84 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
+import  Inputfelt, { texts } from '../../js/skjema/Inputfelt';
+  
+
+const inputRole = 'input';
+const submitRole = 'submit';
+const alertRole = 'alert';
+
+test('Skal vise riktig feilmeldinger ved validering', () => {
```

Comment:
Må gjerne konvertere denne til typescript, og legge til typescript-støtte for react-testing-library/jest. Her er en liten how-to: https://www.pluralsight.com/guides/how-to-test-react-components-in-typescript

Reply:
Takk for tipset! :) Jeg skiller det ut i en egen oppgave 👍